vision/cloud-client/detect/docpdf.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""OCR with PDF/TIFF as source files on GCS
+
+Example:
+    python docpdf.py --gcs-source-uri gs://python-docs-samples-tests/HodgeConj.pdf \
```

Comment:
Personal wish: file name change. detect_beta or detect_pdf

Reply:
+1 let's make this more conventional with the other CLIs, I like detect_pdf

Reply:
changing to detect_pdf.py